queries/__init__.py

```diff
@@ -52,17 +52,17 @@
 from pytz import country_timezones
 
 from geo import country_name_for_isocode, iceprep_for_cc, LatLonTuple
-from queries.num import number_to_text, float_to_text
+from queries.utility.number_utils import number_to_text, float_to_text
```

Comment:
Suggestion: queries/utility -> queries/util and queries.utility.number_utils -> queries.util.num or queries.util.numbers
Good to keep naming nice and short and non-redundant.

Reply:
Good point, I'll shorten the names :)

Reply:
This causes confusion with `util.py` in the root. Is it ok if I rename that file to `utility.py`?

Reply:
Does it, though? The root level util is global, whereas queries.util is utilities for queries. Seems pretty clear to me. But do what you feel is best. The main point is to keep nested hierarchies, files and folder names short to avoid horrible Java-style deep nested hierarchies with excessively verbose names.

Reply:
It at least confuses the linter when I do `from util import GREYNIR_ROOT_PATH, read_api_key` in `queries/__init__.py` because we always import as if were in the root, but the util folder is also in the queries folder. I also tested importing from `..util` but then it complains about not having a parent module :/.

Reply:
OK, name it as you like, we wouldn't want to confuse the linter :).

Reply:
Shortened some of the file/folder names :+1:.